### DISCUSSION TITLE
Use Grouped Multi-Select fields as Kanban Columns instead of just Status

### DISCUSSION THREAD

**Discussion by neverether** [community/community#5571] [Projects and Issues]
It's really quite limiting how Kanban is implemented in the current beta.

One particular use case this could come in handy is a Bug Tracker view which I have set up a new view to show the label `bug`:

Currently in Kanban view, we are only able to sort and move issues against the built-in `Status` field (To-Do, etc)

It would be fantastic if the columns would work with any Multi-Select grouping – in my case Group by `Priority(Multi-Select)` so that I can assign priority to bugs in a specific view, making it much easier to use the projects view for triage

**Comment by HonzaMikula** [comment]
I totally agree. E.g. we are using [Shape Up methodology](https://basecamp.com/shapeup). Therefore we have three types of projects:
* Mini-projects (something like epic)
* Chunk of works (something like user stories)
* Dev tasks (sub tasks)

And each type has little bit different flow and therefore statuses. Dev task have classic To Do, In Progress, Done. But e.g. Chunk of works have Not started, Unknown, Known, and Done. Mini-projects have In Shaping, Ready for Betting Table, Selected for Cycle, Done.

In the current setup, you cannot distinguish those things. But otherwise I really love new Project boards, so thanks for the great work. 👏

**Comment by jesfer** [comment]
You can do that in the table view... Group by whatever custom field you want. Then drag'n'drop items without a value to the desired group. Value is changed to that of the group.

Another way is to filter the table to only list items without a field value (I think GH implemented that now) with a filter like "no:Priority" and then simply assign a value to them.

**Reply by HonzaMikula** [reply]
Yeah, that's nice. But I would like to have the possibility to have custom columns in the Board view.
